core/trie/node.go

```diff
@@ -109,9 +110,17 @@ func (n *Node) UnmarshalBinary(data []byte) error {
 		var pathP **bitset.BitSet
 		switch head {
 		case 'l':
-			pathP = &(n.left)
+			if n.left == nil {
+				pathP = &(n.left)
+			} else {
+				return ErrMalformedNode{"multiple left childs are not support"}
```

Comment:
```suggestion
				return ErrMalformedNode{"multiple left children are not supported"}
```

Reply:
Sorry for make so many spell error.

Reply:
No worries! This is what reviews are for.